src/js/jquery.pdbox.js

```diff
@@ -104,10 +106,23 @@ $.pdBox = (function () {
 			this.options[optName] = undefined;
 		}
 
+		this.addEventListener('afterOpen', function () {
+			this.trap = focusTrap.createFocusTrap('.pdbox', {
+				escapeDeactivates: false,
+				clickOutsideDeactivates: false
+			});
+			this.trap.activate();
+		});
+
 		this.addEventListener('load', function () {
 			this.rootElem.removeClass('pdbox--loading');
+			this.trap.updateContainerElements('.pdbox');
```

Comment:
Pokud by to fungovalo (mělo by), tak takto:
```suggestion
			this.trap.updateContainerElements(this.rootElem);
```
(Případně jiný element, ale měl by to být rootElem, viz odebírání class výše.) Díky tomu se nebude zbytečně ten element vyhledávat, předá se přímo reference.